search/v2/geocoder.cpp

```diff
@@ -1092,6 +1191,33 @@ void Geocoder::FindPaths()
   });
 }
 
+unique_ptr<coding::CompressedBitVector> Geocoder::LoadCategories(
+    MwmContext & context, vector<strings::UniString> const & categories)
+{
+  ASSERT(context.m_handle.IsAlive() && HasSearchIndex(context.m_value), ());
```

Comment:
Please, separate this complex ASSERT() to two assertions. In the case of ASSERTion failure it's unclear what part of conjuction is false.